### ISSUE TITLE
Call to undefined method Parse\ParseClient::setServerURL()

### ISSUE THREAD

**Issue by petterfeed** [ParsePlatform/parse-php-sdk#255]
Trying to migrate my parse server over to heroku.

But I have some problems with the PHP SDK. I run the latest version of the SDK and PHP v 5.6.10. 

When adding ParseClient::setServerURL('https://my-parse-server.com','parse') it gives me a Fatal error: Call to undefined method Parse\ParseClient::setServerURL()

This is my setup:

```
require 'lib/parse-php-sdk-master/autoload.php';

use Parse\ParseCloud;
use Parse\ParseClient;
use Parse\ParseSession;

$my_app_id = '****';
$my_rest_key = '****';
$my_master_key = '****';

ParseClient::initialize( $my_app_id, $my_rest_key, $my_master_key );
ParseClient::setServerURL('https://****.herokuapp.com','parse');
```

Any ideas?

**Comment by sanjibnarzary** [comment]
I got the same problem while trying to connect ParsePlatform from a Laravel Parse client in Windows Machine

`ErrorException in ParseClient.php line 148:
Missing argument 2 for Parse\ParseClient::setServerURL(), called in C:\Users\user\Documents\TestParse\routes\web.php on line 16 and defined`

My web.php contents is as follows

`Route::get('/', function(){
    \Parse\ParseClient::initialize('aid', '', 'mk');
    \Parse\ParseClient::setServerURL('http://localhost:1337/parse');
    $testObject = \Parse\ParseObject::create("TestObject");
    $testObject->set("foo", "bar");
    $testObject->save();
});`

**Comment by montymxb** [comment]
@sanjibnarzary you're probably looking for the solution outlined in #247, the problem there was dealing with a Laravel package as well. 

As for the original issue I'm not too sure. I would double check everything in your parse php sdk and make sure it is up to date. Particularly I would check that your ParseClient.php has the newest definition of ```setServerURL``` by inspecting it directly, perhaps something is amiss there?

**Comment by sanjibnarzary** [comment]
In my XAMPP server without Laravel the same code is working with a little modification in setServerUTRL(). Here is the working code but the same code is not working with Laravel. I tried with LaraParse as well without LaraParse only with ParsePHPSDK still not working.

Buth the following code working in a XAMPP server with Parse Platform installed locally

`<?php
require 'parse-php-sdk/autoload.php';

\Parse\ParseClient::initialize('aid', '', 'mk');

\Parse\ParseClient::setServerURL('http://localhost:1337','parse'); //Here I modified

$testObject = \Parse\ParseObject::create("TestObject");

$testObject->set("foo", "bar");

$testObject->save();
`
